### PR TITLE
Fixed #192 #196

### LINKED ISSUE #192: Fix Footer at the bottom of the screen
Please see the below screenshot: 

screen shot 2017-02-12 at 10 59 07 pm 2

### LINKED ISSUE #196: Table Borders not visible correctly
For reference, see this 
screen shot 2017-02-22 at 5 20 40 am

### PR DESCRIPTION
**Pull request by singhsterabhi**
fixed #192
fixed #196

### CODE REVIEW

File: HTTP_200/settings.py [hunk 1]
```diff
@@ -107,23 +107,23 @@
```
Comment (RishabhJain2018) [inline HTTP_200/settings.py]: @singhsterabhi I think you have committed this file by mistake. Please remove it as this isn't the part of this PR.

File: requirements.txt [hunk 2]
```diff
@@ -12,7 +12,7 @@ django-braces
 django-autofixture
```
Comment (RishabhJain2018) [inline requirements.txt]: Also , please remove this file too and create a new PR for updating the dependencies.

**Comment by RishabhJain2018** [comment]
Moreover, please update your review comment by using these guidelines https://help.github.com/articles/closing-issues-using-keywords/

**Comment by RishabhJain2018** [comment]
@singhsterabhi The code looks good to me. Can you please paste the screenshot for before and after the change.

**Comment by singhsterabhi** [comment]
wifi form
before:

after:
screenshot from 2017-09-12 18-12-14
Similarly on pages index, about, contact, profile,etc.

**Comment by RishabhJain2018** [comment]
@singhsterabhi Can you please open a new PR for #196 . It would make the code more modular.

**Comment by RishabhJain2018** [comment]
@singhsterabhi Please let me know if you are facing any problem. I will be glad to help you!

**Comment by RishabhJain2018** [comment]
@singhsterabhi let's discuss it tomorrow and get it merged :)

**Comment by singhsterabhi** [comment]
Created new pr #226

**Pull request closed without merging**